flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/cli/SqlCommandParser.java

```diff
@@ -132,11 +132,10 @@ private SqlCommandParser() {
 			cmd = SqlCommand.DROP_CATALOG;
 		} else if (operation instanceof UseCatalogOperation) {
 			cmd = SqlCommand.USE_CATALOG;
-			operands = new String[] { String.format("`%s`", ((UseCatalogOperation) operation).getCatalogName()) };
+			operands = new String[] { ((UseCatalogOperation) operation).getCatalogName() };
 		} else if (operation instanceof UseDatabaseOperation) {
 			cmd = SqlCommand.USE;
-			UseDatabaseOperation op = ((UseDatabaseOperation) operation);
-			operands = new String[] { String.format("`%s`.`%s`", op.getCatalogName(), op.getDatabaseName()) };
+			operands = new String[] { ((UseDatabaseOperation) operation).getDatabaseName() };
```

Comment:
```suggestion
			operands = new String[]{String.format("%s.%s", StringUtils.strip(op.getCatalogName(), "`"), StringUtils.strip(op.getDatabaseName(), "`"))};
```

Reply:
catalog name is unnecessary, only database is needed.